java/test/org/openqa/selenium/chrome/ChromeDriverFunctionalTest.java

```diff
@@ -71,6 +71,18 @@ public void builderOverridesDefaultChromeOptions() {
     assertThat(localDriver.manage().timeouts().getImplicitWaitTimeout()).isEqualTo(Duration.ofMillis(1));
   }
 
+  @Test
+  @NoDriverBeforeTest
+  public void driverOverridesDefaultClientConfig() {
```

Comment:
@code-with-abdullah - Can we please refactor the test to look something like this ? 

```java
    org.assertj.core.api.Assertions.assertThatThrownBy(() -> {
      ClientConfig clientConfig = ClientConfig.defaultConfig().readTimeout(Duration.ofSeconds(0));
      localDriver = new ChromeDriver(ChromeDriverService.createDefaultService(),
        new ChromeOptions(), clientConfig);
    }).isInstanceOf(SessionNotCreatedException.class);
```

This is how the other tests in this class that expect exceptions have been written. So we will stick to the same approach. Please have the other tests also refactored on the same lines (Also please check for un-used imports in the files that you changed and kindly help get them removed as well)

Apart from this test case related observation everything else looks good.

Reply:
Great observation @krmahadevan. Will update and push as soon as possible.